cpp/tests/test_state_age_function.cpp

```diff
@@ -204,16 +229,18 @@ TEST(TestStateAgeFunction, testSAFWrapperSettersAndGetters)
 
     EXPECT_EQ(wrapper.get_parameter(), testvalue_before);
     EXPECT_EQ(wrapper.get_support_max(dt), testvalue_before);
+    EXPECT_EQ(wrapper.get_mean(dt), testvalue_before * 0.5);
```

Comment:
A comment that this is the formula for the mean for SmootherCosine would be helpful here (although this is mentioned in the comment for get_mean in SmootherCosine).